src/features/movie/components/cast/Cast.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { screen, fireEvent, render, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { MemoryRouter } from "react-router-dom";
+import { expectNever } from "test/utilities/utilities";
+import { Cast } from "./Cast";
+import { movie } from "fakeData/data";
+
+describe("Cast", () => {
+  it("should render Cast", () => {
+    const { getByText } = render(
+      <MemoryRouter initialEntries={[{ pathname: "/movie/:id" }]}>
```

Comment:
Why does a leaf component like "Cast" need to know about navigation? You can check if a component renders without coupling it to a concept of navigation.

Reply:
I will remove it.